.github/workflows/main.yml

```
@@ -402,11 +402,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          # We need accurate timestamps to avoid prettier cache invalidation
```

I didn't immediately draw the connection between this and `git restore-mtime`.

```suggestion
          # We need accurate timestamps to avoid prettier cache invalidation.
          # We'll use `git restore-mtime`, which requires the full history.
```